### PR TITLE
Feature/add first post issue 3

### LINKED ISSUE #3: Add post that announces a new podcast episode
title

### PR DESCRIPTION
**Pull request by min-dles**
- edited a sample post for details re: upcoming Gyst Rewind episode
- added a sample post of an existing Gyst Rewind episode 
- this initial PR should resolve #3

### CODE REVIEW

**Comment by min-dles** [comment]
### <span aria-hidden="true">✅</span> Deploy Preview for *stately-mochi-93d6d5* ready!


|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 26f1eac104d14f86ccb98612413f4a9760c96191 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/stately-mochi-93d6d5/deploys/64e53fa320cdfb00082d7300 |
|<span aria-hidden="true">😎</span> Deploy Preview | [https://deploy-preview-7--stately-mochi-93d6d5.netlify.app](https://deploy-preview-7--stately-mochi-93d6d5.netlify.app) |
|<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
---

_To edit notification comments on pull requests, go to your [Netlify site configuration](https://app.netlify.com/sites/stately-mochi-93d6d5/configuration/deploys#deploy-notifications)._

File: pages/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default function Index({ posts, globalData }) {
 
   return (
     <Layout>
-      <div className={styles.wrapper}>
+      <div className={`${styles.wrapper} before:bg-white before:bg-white before:backdrop-blur-lg before:dark:bg-black before:dark:bg-opacity-30 before:bg-opacity-10 `}>
         <DynamicReactPlayer
```
Comment (GirasolXimena) [inline pages/index.js]: We should be only adding the post, not changing styles so we don't interfere with the other ticket

File: styles/player.module.css [hunk 2]
```diff
@@ -20,7 +20,8 @@
     right: 0;
     bottom: 0;
     /* Transition from a solid color at the top to transparent at the bottom */
-    background: linear-gradient(to top, rgba(255, 255, 255, 0.99), rgba(255, 255, 255, 0.75));
+    /* background: linear-gradient(to top, rgba(255, 255, 255, 0.99), rgba(255, 255, 255, 0.75)); */
+    opacity: var(--tw-bg-opacity);
```
Comment (GirasolXimena) [inline styles/player.module.css]:
```suggestion
background: linear-gradient(to top, rgba(255, 255, 255, 0.99), rgba(255, 255, 255, 0.75));
```
same as before

File: pages/index.js (hunk 1 above)
Comment (GirasolXimena) [reply pages/index.js]:
```suggestion
      <div className={styles.wrapper}>
```

File: posts/new-ep-sat.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: New Ep Drops Sat
+description: "Check out Gyst Rewind this Saturday 8.19.23 for an excellent conversation with B.I.G.G (@biggstreetz on IG) "
+date: Aug 17 2023
+---
+
+New episode with [@biggstreetz](https://www.instagram.com/biggstreetz/) dropping Saturday! [#podcast](https://www.instagram.com/explore/tags/podcast/?hl=en) [#Gystrewind](https://www.instagram.com/gyst.rewind/) [#Minneapolis](https://www.instagram.com/explore/locations/214063175/minneapolis-minnesota/?hl=en)
```
Comment (GirasolXimena) [inline posts/new-ep-sat.mdx:7]:
```suggestion
New episode with <a href="https://www.instagram.com/biggstreetz/" target="_blank">@biggstreetz</a> dropping Saturday! [#podcast](https://www.instagram.com/explore/tags/podcast/?hl=en) [#Gystrewind](https://www.instagram.com/gyst.rewind/) [#Minneapolis](https://www.instagram.com/explore/locations/214063175/minneapolis-minnesota/?hl=en)
```

see if this works?

got it from [here](https://www.eddymens.com/blog/how-to-make-a-markdown-link-open-in-another-tab#:~:text=You%20just%20add%20the%20target,in%20a%20different%20browser%20tab)

**Review by GirasolXimena — CHANGES_REQUESTED** [review]

File: posts/new-ep-sat.mdx (hunk 3 above)
Comment (min-dles) [reply posts/new-ep-sat.mdx:7]: we can try this later; i know that MDX works with the current format. it may need to be updated tho for best practices

**Comment by min-dles** [comment]
hi it's me not me don't forget

**Comment by min-dles** [comment]
nice job me

**Pull request closed without merging**